mergify/README.md

```diff
@@ -10,13 +10,29 @@ and optimize your development workflow using this Datadog integration.
 
 ## Setup
 
-### Installation
+You can either install the built-in integration, or install the agent.
 
-#### From release
+### Setup Built-In Integration
+
+- **In Mergify dashboard**: Go to [Mergify dashboard][10], log in and navigate to **Integrations**.
+
+- Click the **Connect** button in the **Datadog** tile.
+
+![Mergify Screenshot][12]
+
+- You will be redirected to the Datadog authorization page.
+
+- Log in to Datadog to complete setup.
+
+Your Mergify statistics now appear in Datadog.
+
+### Setup Agent
+
+#### From Release
 
 Run `datadog-agent integration install -t datadog-mergify==<INTEGRATION_VERSION>`.
```

Comment:
~~~suggestion
To set up the agent from release, run the following command, replacing `<INTEGRATION_VERSION>` with the version number for this integration:

```
datadog-agent integration install -t datadog-mergify==<INTEGRATION_VERSION>
```
~~~